ios/FluentUI/Core/FluentUIHostingController.swift

```diff
@@ -40,16 +40,71 @@ extension UIView {
 }
 
 /// FluentUI specific implementation of the UIHostingController which adds a workaround for disabling safeAreaInsets for its view.
```

Comment:
Should probably mention this is now required for automatic fluent theme passthrough.

Reply:
Yep, I'll do a quick pass on the comments in this file post-refactor.

Reply:
Also renaming the class to `FluentThemedHostingController` to be more explicit about its primary goal.